community.html

```diff
@@ -18,38 +18,50 @@
             <div class="row">
                 <div class="col-sm-6 col-md-12 community-section">
                     <div class="col-md-2">
-                        <img src="assets/mail-list.svg" class="com-image" alt="mailing list icon">
+                        <img src="assets/github.svg" class="com-image" alt="github icon">
                     </div>
                     <div class="col-md-8 community-text">
-                        <h2>Jupyter General Mailing List</h2>
-                        <p>A Google Group for general Jupyter discussions of Jupyter's use.</p>
+                        <h2>Jupyter on <br id="github-break">GitHub</h2>
+                        <p>GitHub is our primary place to report issues with Jupyter.</p>
                     </div>
                     <div class="col-md-2 community-button">
-                        <a href="https://groups.google.com/forum/#!forum/jupyter">VIEW</a>
+                        <a href="https://github.com/jupyter/help">VIEW</a>
+                    </div>
+                </div>
+                <div class="col-sm-offset-3 col-sm-6 col-sm-offset-3 col-md-offset-0 col-md-12 community-section">
```

Comment:
Ah, I see it too... well the Stack--overflow section is typically something that shouldn't be this way.